src/ckeditor.js

```diff
@@ -9,153 +9,173 @@ import { debounce } from 'lodash-es';
 
 const INPUT_EVENT_DEBOUNCE_WAIT = 300;
 
-export default {
-	name: 'ckeditor',
+/**
+ * Vue component creator.
+ *
+ * @param {Object} options
+ * @param {Boolean} options.isNextVue Informs if current Vue version is the next one (v3.x) or the previous one (v2.x).
```

Comment:
Let's rename `options.isNextVue` to `options.isVue3` to avoid future confusion when Vue 4 is out (or even Vue 3.5).

Reply:
Done.